config/locales/fil/navigation.fil.yml

```diff
@@ -0,0 +1,11 @@
+fil:
+  layouts:
+    navigation:
+      toggle-navigation-button-label: 'I-toggle ang navigation'
+      submit-a-new-restroom-hyperlink-label: 'Magsumite ng Bagong Silid'
```

Comment:
I'm not sure "submit a new room" ("Magsumite ng Bagong Silid") captures the original meaning? We mean only to have people add "restrooms" (or as some would call them: water closets (WC), bathrooms, toilets, washrooms, lavatories).

Adding "a room" (Silid) seems a little too general-purpose? Maybe I am misinterpreting, due to not knowing Filipino, sorry if that's the case.